### PR TITLE
enhance: adjust parquet reader type check with null type

### PR DESCRIPTION
**Pull request by smellthemoon**
#36252 
remove no need type check. if users use null type writer to write parquet, hope it successfully.

### CODE REVIEW

**Comment by smellthemoon** [comment]
related with #36252

**Comment by smellthemoon** [comment]
rerun ut

**Comment by smellthemoon** [comment]
rerun ut

**Comment by smellthemoon** [comment]
rerun ut

File: internal/util/importutilv2/parquet/field_reader.go [hunk 1]
```diff
@@ -353,8 +359,12 @@ func ReadNullableIntegerOrFloatData[T constraints.Integer | constraints.Float](p
 			for i := 0; i < dataNums; i++ {
 				data = append(data, T(float64Reader.Value(i)))
 			}
+		case arrow.NULL:
+			// the chunk type may be *array.Null if the data in chunk is all null
+			validData = append(validData, make([]bool, dataNums)...)
+			data = append(data, make([]T, dataNums)...)
 		default:
-			return nil, nil, WrapTypeErr("integer|float", chunk.DataType().Name(), pcr.field)
+			return nil, nil, WrapTypeErr("integer|float or null", chunk.DataType().Name(), pcr.field)
```
Comment (bigsheeper) [inline internal/util/importutilv2/parquet/field_reader.go:367]: please help to format this to `integer or float or null` or `integer|float|null`

File: internal/util/importutilv2/parquet/util.go [hunk 2]
```diff
@@ -52,10 +52,10 @@ func CreateFieldReaders(ctx context.Context, fileReader *pqarrow.FileReader, sch
 		return nil, merr.WrapErrImportFailed(fmt.Sprintf("get parquet schema failed, err=%v", err))
 	}
 
-	err = isSchemaEqual(schema, pqSchema)
-	if err != nil {
-		return nil, merr.WrapErrImportFailed(fmt.Sprintf("schema not equal, err=%v", err))
-	}
+	// err = isSchemaEqual(schema, pqSchema)
```
Comment (bigsheeper) [inline internal/util/importutilv2/parquet/util.go:55]: can we keep this check and check by arrow.Null type?

**Comment by smellthemoon** [comment]
rerun go-sdk

**Review by congqixia — APPROVED** [review]
/approve

**Pull request merged**